src/routes/routes.tsx

```diff
@@ -21,7 +21,9 @@ const BaseRouter = (props: any) => (
         <AppRoute exact path='/peers/' layout={ Dashboard } component={Peers}/>
         <AppRoute exact path='/accounting/' layout={ Dashboard } component={Accounting}/>
         <AppRoute exact path='/settings/' layout={ Dashboard } component={Settings}/>
+        {/* A catch-all because electron build initial load is: /absolute/path/index.html */}
+        <Route path='*'><Redirect to='/'/></Route>
```

Comment:
If the goal is to have a 'desktop' like application experience, a "not found" doesn't make much sense (since it's not a client-server relationship). There are no browser-back buttons in the electron build either, so if a user were to hit a 404 page they may get stuck to the point of having to exit the app.

There was another reason I added this, though, but the <base> change I made below may have solved the issue as well. Let me investigate more.

Reply:
@nugaon I looked into the other reason I added this, and it's because when electron first loads, the path it starts is an absolute path to the location of the application, and then its asar file (a bundle of web assets) and _then_ the location of the index.html file that's built from React.

If the electron had a browser bar, it's location would be something like : `file://home/andy/.config/bee-status/package/app.asar/build/index.html`

Which obviously doesn't match the `exact path='/'` of the first route. I add the catch-all to redirect to the `/` so that the home dashboard loads properly.

Before this addition, the app would start and it'd just be stuck on a blank screen because the route matched nothing, so nothing loaded.